lib/handlers/quote/schema.ts

```diff
@@ -21,6 +21,7 @@ export const PostQuoteRequestBodyJoi = Joi.object({
     }),
   swapper: FieldValidator.address.optional(),
   useUniswapX: Joi.boolean().default(false).optional(),
+  sendPortionEnabled: Joi.boolean().default(false).optional(),
```

Comment:
Note - `portion` from [QuoteRequestInfo](https://github.com/Uniswap/unified-routing-api/blob/main/lib/entities/request/index.ts#L21C18-L21C34) is intentionally not checked here. 

The reason is because `portion` was not sent from the upstream clients, but it was populated during [handleRequest](https://github.com/Uniswap/unified-routing-api/blob/main/lib/handlers/base/api-handler.ts#L162). The request schema check happened before [handleRequest](https://github.com/Uniswap/unified-routing-api/blob/main/lib/handlers/base/api-handler.ts#L162), i.e. within [parseAndValidateRequest](https://github.com/Uniswap/unified-routing-api/blob/main/lib/handlers/base/api-handler.ts#L309). Checking `portion` schema would not be meaningful, since it's always undefined within [parseAndValidateRequest](https://github.com/Uniswap/unified-routing-api/blob/main/lib/handlers/base/api-handler.ts#L309)